ignite/tests/compose/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  ignite:
+    container_name: dd-ignite
+    image: apacheignite/ignite
+    environment:
+     - CONFIG_URI=file:///config.xml
+    ports:
+     - 49112:49112
+    volumes:
+     - ${CONFIG_FILE}:/config.xml
+     - ${FUNCTIONS_FILE}:/opt/ignite/apache-ignite/bin/include/functions.sh
```

Comment:
Any reason not to simplify to `./file:...`?

Reply:
Probably not :)